### DISCUSSION TITLE
Is OpenAudible still fully open source?

### DISCUSSION THREAD

**Discussion by trymeouteh** [openaudible/openaudible#659] [Q&A] [answered]
Is OpenAudible still fully open source or is the last version that was fully open source was 1.1.8?

**Comment by Dev-kris** [comment]
Looks like it isn't. Despite the name and the .org address it appears to be just another commercial product with desirable features locked behind a license. 

I don't mind this really, and would have been up for buying it except for the feeling that it's preying on the assumption of something it's not.

**Comment by openaudible** [comment, marked as answer]
OpenAudible is no longer open source. But it has the most features of any audiobook utility we know of! 
Our original donation system got about $50/week and that didn't cover the time and expense needed to make it a great product.

**Reply by izzy** [reply]
Well, what you have now isn't a "great product" either. You've literally just put a price tag on an open source app.

I would've been willing to take up with the bad UI/UX from something that's free but not for a paid product...

**Reply by charltonstanley** [reply]
> OpenAudible is no longer open source. But it has the most features of any audiobook utility we know of! Our original donation system got about $50/week and that didn't cover the time and expense needed to make it a great product.

It'd be great if you were re-branded to something that didn't give the wrong impression. Not saying that you have to re-brand to "ClosedAudible" or anything, but maybe not "Open" anymore?

**Reply by openaudible** [reply]
The word open has several meanings.. We still "open" your audible content to users. 
Open Source almost always means "impoverished developers"... as we found out when we tried to ask for donations.  I like free stuff as much as the next person, but I also like to support small developers who work hard on interesting and useful products.

**Reply by MarqueIV** [reply]
> The word open has several meanings.. We still "open" your audible content to users.
> Open Source almost always means "impoverished developers"... as we found out when we tried to ask for donations. I like free stuff as much as the next person, but I also like to support small developers who work hard on interesting and useful products.

This doesn't really address the commenters question/issue.  You had a product that was open source, you called it 'OpenAudible', then you changed it to closed-source.  As such, you should change the name.  That should be the case even if it didn't have the word 'Open' in it because it is confusing to people, such as myself who was about to pull the 'latest' code, only to find out it isn't the 'latest' code, and now, that it's closed, and from a developer I don't know and who's done something really shunned in the open source community--switched to paid--remember... being open, you weren't the only contributor, no matter how small a contribution--it's not exactly something I'm confident in trusting anymore.

IMHO, you should have renamed the product to something else.  You could even have said 'Based on my earlier work in OpenAudible' or something like that.  But the way it is now is misleading, and by your 'open has several meanings' stance, intentionally so.  After all, 'Audible' too has several meanings.  Do you now support any audio file anywhere from any store because all audio is by definition 'Audible'?  Yes, I'm purposely being obtuse, but that's because again, this kind of 'Was-open-now-closed' without changing the name is deceitful.

Likewise, you should also change to OpenAudible.com.  Leave Org referring to the Open Source version, then let them redirect by clicking on a link explaining the change.  Again, it's deceiving/deceitful to use 'org' for commercial purposes when there's a history of it first being open.

Now... as a dev who gets paid for a living, I 100% support your desire to charge for your work.  I encourage it actually. As you said, Open Source is really 'I'm not getting paid either!' work and it does take a lot of time.  That's why I want to be completely clear that I have no problem with people who suddenly decide to start charging for their work.  But... a) you need to make sure what you're now charging for contains **zero** code from anyone besides yourself who would have contributed to the product (I myself got pissed at a repo holder when I contributed open-source changes that they later closed.  I contemplated suing actually, but ultimately decided not to), and two, you make a **clear demarcation** between the open- and closed varieties.  Again, get paid!  You deserve it.  But not on the backs of open-source work.

Anyway, that's just my 2c upgraded to $1.43 for inflation.

**Comment by openaudible** [comment]
There was no profiting from the backs of open source contributors.  

That was the problem— no one stepped up and submitted any code or new features. It was all written in house by our small team. You can check the pull requests. 

Our only mistake was giving open source a try. Sadly it is a community of petty complainers, freeloaders and lawsuit seekers. See above. 

The exception for us is ffmpeg. We use a legal LGPL version in our software. They deserve a huge thanks for their decades of work… and we have made financial donations to the ffmpeg organization and encourage everyone who enjoys ffmpeg to do the same. They do thankless work and I’m guessing fewer than 1 in 1000 users pay a penny to them.

**Reply by MarqueIV** [reply]
As I said before, I 100% support you being paid for your work, especially when you are writing software that's entire purpose is to help people manage their paid-for content.  In other words, you're not writing an open-source tool to support other open-source endeavors. Amazon is getting paid, why not you?

The issue is in the name, the domain, and essentially using GitHub as free advertising, starting from an open product.  It just smells funny.  

Again, even saying 'Based on the DNA of our open-source OpenAudible effort... bla bla...' would be understandable, but more importantly, clear.

So again, I get it. And I agree... open source is hard and thankless work whether a developer, maintainer or even just providing support dealing with a-holes like me leaving comments like this.  You created a useful product.  You absolutely should be paid.

Just change the name and domain or else people such as myself and the other posters above will continue to have a bad taste in our mouths over this.

Let me put it another way... had I seen what I was suggesting... 'AudibleManager... from the creators of OpenAudible' or 'Based on the DNA of our OpenAudible open-source effort' or something like that would've made me excited for the product because I'd think 'It got so good, they cleaned it up and commercialized it!  Especially if you did something like leaving the 'guts' of it (i.e. the part that actually does the downloading) still open source here, so we could leverage your 'engine' with our own UIs or scripting or whatever.  But the way it is now... it just 'feels' wrong.  Hope that makes sense.

I do wish you well tho.  It's good to be rewarded for your hard efforts, especially when again, you're supporting someone else's commercial endeavor.

**Reply by izzy** [reply]
I mean, it's not just the misleading "Open" part of the name. It's also the Audible part. Even with freeware a trademark-infringing name is shoddy at best, but as a commercial solution @openaudible is in for a ride if Amazon's lawyers have a bad day and decide to take aim at it. I mean, the name's just giving them a real easy way to take down a software(and it's dev) that's probably not in their capitalist interest anyway...

No matter from what angle you look at it, changing the name to something wholly different and doing what @MarqueIV suggested(i.e. a mention of what it's built on) would probably be in everyone's best interest.

**Reply by mzryz** [reply]
I agree - I installed OpenAudible through the AUR on my distro, thinking I was installing an open source product. I then went through the process of allowing the app to connect to my audible account and access my books - I then discovered it said 'License: Demo' in the main UI. I was suddenly shocked that I'd provided my Amazon login details to a closed-source app and felt instantly quite violated in that regard. I want to use open-source software, I trust it since it's auditable and you can confirm your data is safe, but now I've logged into a closed source app and given my data away, with no idea if it's safe.

A perfect example of the name & domain being very misleading.

**Reply by mkb79** [reply]
@mzryz 
To make sure, no one can use your credentials I would suggest to change your password. Openaudible will create a dummy Audible device. I would suggest, to remove the device on the Amazon website (e.g. https://www.amazon.com/hz/mycd/digital-console/alldevices). After removing the device, the credentials, which are given via the registration, are useless for everyone. With one exception. The provided access token can't be revoked. They are valid until it expires (max. 60 minutes)!

**Comment by pierow2k** [comment]
I just went through the same process as others. I found the repo while searching, read through the `README`, it says that the software in the repo isn't the latest version _(confusing)_, went to the website, and it mentions paying for a license.  I then returned to the repo, found that there isn't _any_ software hosted on GitHub, read through old issues, and found this thread in Discussions.

The use of `Open` in the name of the repo is misleading. Yes, you can `Open` Audible files but I think we all know that isn't the typical interpretation of `Open` in the context of GitHub.  There's also nothing of value in the repository - it only serves to advertise your _commercial_ product and point visitors to a website where that software can be purchased.

**Comment by krikey** [comment]
+1 to considering renaming your product as its clearly not "open" in the Open Source spirit that I believe should be followed.

I too installed OpenAudible only to be presented with the "demo" account and linking it to my amazon account.

I don't buy the "The word open has several meanings.. We still "open" your audible content to users." as I wonder if that's how it started off meaning and feel like a cheap trick.

As mentioned, developers like yourselves should be remunerated or rewarded in some form for their time spent, but re-purposing a previously open source product (even if its your own) without changing its name has left me feeling a little annoyed.

Please consider rebranding your product gradually as to not lose out on sales of course to something that doesn't suggest Open Source.

**Comment by olavlan** [comment]
I agree with the others, but if you're unwilling to change the name, I think it would be in your best interest to add a clear disclaimer at the top of the README stating that the source code is now closed and which features have been commercialized. It would make the project look more professional and transparent—something that could have even influenced my decision to buy it. People would really appreciate knowing this upfront rather than feeling like their time was wasted.

**Comment by AskAlice** [comment]
rename this repo this is ridiculous. open implies that it is FOSS. this could lead to a false advertising liability

**Comment by akumaburn** [comment]
Absolutely, What's the point of releasing old source code alongside your binary releases as if that is the source code for that version? Totally misleading!
